docs/usage.md

```diff
@@ -124,52 +168,76 @@ It's a good idea to specify a pipeline version when running the pipeline on your
+
+If `-profile` is not specified, the pipeline will run locally and expect all software to be installed and available on the `PATH`. This is _not_ recommended.
 
-- `awsbatch`
-  - A generic configuration profile to be used with AWS Batch.
-- `conda`
-  - A generic configuration profile to be used with [conda](https://conda.io/docs/)
-  - Pulls most software from [Bioconda](https://bioconda.github.io/)
 - `docker`
   - A generic configuration profile to be used with [Docker](http://docker.com/)
   - Pulls software from dockerhub: [`nfcore/sarek`](http://hub.docker.com/r/nfcore/sarek/)
 - `singularity`
-  - A generic configuration profile to be used with [Singularity](http://singularity.lbl.gov/)
+  - A generic configuration profile to be used with [Singularity](https://sylabs.io/docs/)
   - Pulls software from DockerHub: [`nfcore/sarek`](http://hub.docker.com/r/nfcore/sarek/)
+- `conda`
+  - Please only use Conda as a last resort `i.e.` when it's not possible to run the pipeline with Docker or Singularity.
+  - A generic configuration profile to be used with [conda](https://conda.io/docs/)
+  - Pulls most software from [Bioconda](https://bioconda.github.io/)
 - `test`
   - A profile with a complete configuration for automated testing
   - Includes links to test data so needs no other parameters
+- `test_annotation`
+  - A profile with a complete configuration for automated testing
+  - Includes links to test data so needs no other parameters
+- `test_no_gatk_spark`
+  - A profile with a complete configuration for automated testing
+  - Includes links to test data so needs no other parameters
+- `test_split_fastq`
+  - A profile with a complete configuration for automated testing
+  - Includes links to test data so needs no other parameters
+- `test_targeted`
+  - A profile with a complete configuration for automated testing
+  - Includes links to test data so needs no other parameters
+- `test_tool`
+  - A profile with a complete configuration for automated testing
+  - Includes links to test data so needs no other parameters
+- `test_trimming`
+  - A profile with a complete configuration for automated testing
+  - Includes links to test data so needs no other parameters
 
 ### --input
 
-Use this to specify the location of your input TSV file, on `mapping`, `recalibrate` and `variantcalling` steps.
-For example:
+Use this to specify the location of your input TSV file. For example:
+TSV file should correspond to the correct step, see [`--step`](#--step) and [input](input.md) documentation for more information
 
 ```bash
 --input sample.tsv
 ```
 
-Multiple TSV files can be specified if the path must be enclosed in quotes
+Multiple TSV files can be specified if the path must be enclosed in quotes.
 
-Use this to specify the location to a directory on `mapping` step with a single germline sample only.
+Use this to specify the location to a directory with fastq files on `mapping` step for a single germline sample only.
```

Comment:
I don't know what you mean by '**on** mapping step' here

Reply:
That it works only on the `mapping` step

Reply:
Ah ok, the nI would say:
```suggestion
Use this to specify the location to a directory with fastq files for the `mapping` step of single germline samples only.
```